README.md

```diff
@@ -15,7 +15,11 @@ If you're interested in participating in the Community Committee as a member, yo
 
 Community Committee meetings will be broadcast via Google Hangouts, will be announced ahead of time for access, and coordinated to optimize for contributor timezones.
 
-## Current Teams and Working Groups
+## Current Initiatives, Teams, and Working Groups
+
+### Initiatives
+Initiatives are projects that the Community Committee and members of the broader community are collaborating on to enable Node.js across the ecosystem.
+- [User Feedback](https://github.com/nodejs/user-feedback) - **Community Committee contact**: Michael Dawson ([mhdawson](https://github.com/mhdawson))
```

Comment:
Since @mhdawson is currently identified as the CommComm champion for this initiative, I'm comfortable with this landing. We can update as the initiative gets off the ground.